docs/TESTING_WITH_SANITIZERS.md

```diff
@@ -13,6 +13,20 @@ This is mostly done automagically by various CI systems for each PR, but you may
 want to do it locally as well. The process slightly varies depending on the
 compiler you want to use and which part of the test suite you want to run.
 
+## mkosi
+
+To build with sanitizers in mkosi, create a file 20-local.conf in mkosi.default.d/ and add the following
```

Comment:
Since this feature is new I think `mkosi` could be replace with something like `mkosi >= 14 (or built from source)` to make it clear that older versions can't be used to make it work.